serverless.yml

```diff
@@ -8,11 +8,12 @@ useDotenv: true
 
 provider:
     name: aws
+    stackName: ${self:service}
```

Comment:
The default stack name is `${self:service}-${self:provider:stage}`. The `stage` suffix might be confusing in the workshop.